src/bin/generate_test_vector.rs

```diff
@@ -0,0 +1,75 @@
+use color_eyre::eyre::{Result, WrapErr};
+use prio::{
+    field::{Field126, Field32, Field64, Field80, FieldElement, FieldPriov2},
+    test_vector::EnpaTestVector,
+};
+use serde::Serialize;
+use std::str::FromStr;
+use structopt::StructOpt;
+
+#[derive(Debug)]
+enum FieldArgument {
+    FieldPriov2,
+    Field32,
+    Field64,
+    Field80,
+    Field126,
+}
+
+impl FromStr for FieldArgument {
+    type Err = &'static str;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s.to_lowercase().as_str() {
+            "fieldpriov2" => Ok(Self::FieldPriov2),
+            "field32" => Ok(Self::Field32),
+            "field64" => Ok(Self::Field64),
+            "field80" => Ok(Self::Field80),
+            "field126" => Ok(Self::Field126),
+            _ => Err("unknown field"),
+        }
+    }
+}
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "generate-test-vector",
+    about = "Generate test vectors for ENPA",
+    rename_all = "kebab-case",
+    version = env!("CARGO_PKG_VERSION"),
+)]
+struct Options {
+    /// Dimension (number of bins) of the inputs
+    #[structopt(short, long, required = true)]
+    dimension: usize,
+    /// Number of inputs to generate
+    #[structopt(short, long, required = true)]
+    number_of_inputs: usize,
+    /// Field to use to encode inputs
+    #[structopt(short, long, required = true)]
+    field: FieldArgument,
+}
+
+fn generate_and_print_vector<F: FieldElement + Serialize>(options: Options) -> Result<()> {
```

Comment:
```suggestion
fn generate_and_print_vector<F: FieldElement>(options: Options) -> Result<()> {
```

- [ ] delete redundant trait bound